recipe/meta.yaml

```diff
@@ -214,16 +230,15 @@ outputs:
       run:
         - python
         - {{ pin_subpackage('ray-tune', exact=True) }}
-        - atari_py
         - dm-tree
         - gym
         - lz4
-        # until there is a build of opencv-python-headless which does not require
-        # full graphics stack we have to rely on user system having needed
-        #  graphics libraries installed
```

Comment:
I'd like to keep this comment attached to `opencv` until that dependency is removed